ios/Video/RCTVideo.m

```diff
@@ -1490,4 +1557,333 @@ - (NSString *)cacheDirectoryPath {
+                                                                        }
+                                             ];
+                    self.finishLoadingWithError(licenseError);
+                    return false;
+                }
+            } else {
+                NSError *licenseError = [NSError errorWithDomain: @"RCTVideo"
+                                                            code: -1342
+                                                        userInfo: @{
+                                                                    NSLocalizedDescriptionKey: @"Error obtaining DRM License.",
+                                                                    NSLocalizedFailureReasonErrorKey: @"No certificate URL has been found.",
+                                                                    NSLocalizedRecoverySuggestionErrorKey: @"Did you specified the prop certificateUrl?"
+                                                                    }
+                                         ];
+                self.finishLoadingWithError(licenseError);
+                return false;
+            }
+        } else {
+            NSError *licenseError = [NSError errorWithDomain: @"RCTVideo"
+                                                        code: -1343
+                                                    userInfo: @{
+                                                                NSLocalizedDescriptionKey: @"Error obtaining DRM license.",
+                                                                NSLocalizedFailureReasonErrorKey: @"Not a valid DRM Scheme has found",
+                                                                NSLocalizedRecoverySuggestionErrorKey: @"Have you specified the 'drm' 'type' as fairplay?"
+                                                                }
+                                     ];
+            self.finishLoadingWithError(licenseError);
+            return false;
+        }
+        
+    } else {
+        NSError *licenseError = [NSError errorWithDomain: @"RCTVideo"
+                                                    code: -1344
+                                                userInfo: @{
+                                                            NSLocalizedDescriptionKey: @"Error obtaining DRM license.",
+                                                            NSLocalizedFailureReasonErrorKey: @"No drm object found.",
+                                                            NSLocalizedRecoverySuggestionErrorKey: @"Have you specified the 'drm' prop?"
+                                                            }
+                                 ];
+        self.finishLoadingWithError(licenseError);
+        return false;
+    }
+    
+    
+    return true;
+}
+
+- (NSData *)base64DataFromString: (NSString *)string
```

Comment:
I think this function can be avoided, there are native functions to do that
http://iosdevelopertips.com/core-services/encode-decode-using-base64.html